WooCommerce/Classes/POS/Presentation/Reusable Views/POSErrorExclamationMark.swift

```diff
@@ -3,9 +3,14 @@ import SwiftUI
 struct POSErrorExclamationMark: View {
     @Environment(\.dynamicTypeSize) var dynamicTypeSize
 
+    let size: CGFloat
```

Comment:
super nit: with an initializer this could be private

Updated in https://github.com/woocommerce/woocommerce-ios/pull/14856/commits/aeace6a6ccb879afe5dd77489e46e19933d69cb2

```suggestion
    private let size: CGFloat
```